src/Compilers/CSharp/Portable/Binder/Binder_Expressions.cs

```diff
@@ -6324,15 +6324,17 @@ private static void CombineExtensionMethodArguments(BoundExpression receiver, An
             return result;
         }
 
+#nullable enable
         private MethodGroupResolution BindExtensionMethod(
             SyntaxNode expression,
             string methodName,
-            AnalyzedArguments analyzedArguments,
+            AnalyzedArguments? analyzedArguments,
             BoundExpression left,
             ImmutableArray<TypeWithAnnotations> typeArgumentsWithAnnotations,
             bool isMethodGroupConversion,
             RefKind returnRefKind,
-            TypeSymbol returnType)
+            TypeSymbol? returnType)
+#nullable disable
```

Comment:
>disable [](start = 10, length = 7)

Consider restore here instead.

Reply:
Was it too invasive to enable for the entire method?

---
In reply to: [315460746](https://github.com/dotnet/roslyn/pull/38109#discussion_r315460746) [](ancestors = 315460746)

Reply:
I didn't analyze the correct nullability for the other arguments.

Reply:
I don't know.  It wasn't my intent to fix up the body.

---
In reply to: [315460829](https://github.com/dotnet/roslyn/pull/38109#discussion_r315460829) [](ancestors = 315460829,315460746)

Reply:
We should still use `restore` instead of `disable` so that any future changes to the project context aren't weirdly turned off in the middle of files.

---
In reply to: [315923166](https://github.com/dotnet/roslyn/pull/38109#discussion_r315923166) [](ancestors = 315923166,315460829,315460746)